ch_backup/clickhouse/metadata_cleaner.py

```diff
@@ -65,17 +73,41 @@ def clean_tables_metadata(self, replicated_tables: List[Table]) -> None:
             table_macros.update(macros_to_override)
 
             path_resolved = os.path.abspath(replace_macros(table_path, table_macros))
+            full_table_name = f"{table.database}.{table.database}"
+
+            with self._zk_ctl.zk_client as zk_client:
+                # Both paths are already abs.
+                full_table_zk_path = self._zk_ctl.zk_root_path + path_resolved
+
+                if not zk_client.exists(full_table_zk_path):
+                    logging.debug(
+                        "There are no nodes for the replicated table {} with zk path {}",
+                        full_table_name,
+                        full_table_zk_path,
+                    )
+                    continue
+
+                # We are sure that we want to  drop the table from zk.
+                # To force it we will remove it active flag.
+                active_flag_path = os.path.join(
+                    full_table_zk_path, "replicas", self._replica_to_drop, "is_active"  # type: ignore
+                )
+                try:
+                    if zk_client.exists(active_flag_path):
```

Comment:
Is preliminary exist checking needed ?

Reply:
yesterday i thought that better to make one more request to the zk  than to handle the exception. but it doesn't really make any sense, I'll delete it. :)